web/interceptor/util.go

```diff
@@ -0,0 +1,31 @@
+package interceptor
+
+import (
+	"github.com/quickfeed/quickfeed/database"
+	"github.com/quickfeed/quickfeed/qf"
+)
+
+// isValidSubmission returns true if submitting student has active course enrollment or
```

Comment:
` ... if the student or group submitting the original push event has an active course enrollment in the given course.`